ruby/person.rb

```diff
@@ -2,13 +2,14 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rental
 
   def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @id = Random.rand(1..1000)
+    @rental = []
     super()
   end
 
```

Comment:
- [ ] A person has many rentals, You should implement a method for adding rentals like the example in a previous [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). It might look something like this:

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
- Note: The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association